src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcement.tsx

```diff
@@ -5,71 +5,134 @@ import {
   getTextForLanguage,
   useTranslation,
 } from '@atb/translations';
-import {Image, View} from 'react-native';
+import {Image, Linking, StyleProp, View, ViewStyle} from 'react-native';
 import {StyleSheet, useTheme} from '@atb/theme';
 import {ThemeIcon} from '@atb/components/theme-icon';
 import {Close} from '@atb/assets/svg/mono-icons/actions';
 import {PressableOpacity} from '@atb/components/pressable-opacity';
 import {insets} from '@atb/utils/insets';
+import {
+  GenericSectionItem,
+  LinkSectionItem,
+  Section,
+} from '@atb/components/sections';
+import {AnnouncementSheet} from './AnnouncementSheet';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {animateNextChange} from '@atb/utils/animation';
+import {useAnalytics} from '@atb/analytics';
+import {useAnnouncementsState} from '@atb/announcements';
+import Bugsnag from '@bugsnag/react-native';
 
 type Props = {
   announcement: AnnouncementType;
-  onDismiss: (announcement: AnnouncementType) => void;
+  style?: StyleProp<ViewStyle>;
 };
 
-export const Announcement = ({announcement, onDismiss}: Props) => {
-  const style = useStyle();
+export const Announcement = ({announcement, style}: Props) => {
+  const styles = useStyle();
   const {t} = useTranslation();
   const {language} = useTranslation();
   const {theme} = useTheme();
+  const analytics = useAnalytics();
+  const {dismissAnnouncement} = useAnnouncementsState();
+  const {open: openBottomSheet, close: closeBottomSheet} = useBottomSheet();
+
+  const handleDismiss = (announcement: AnnouncementType) => {
+    animateNextChange();
+    dismissAnnouncement(announcement);
+    analytics.logEvent('Dashboard', 'Announcement dismissed', {
+      id: announcement.id,
+    });
+  };
 
   return (
-    <View style={style.container}>
-      <View
-        style={style.content}
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityHint={t(DashboardTexts.announcemens.button.accessibility)}
-      >
-        {announcement.summaryImage && (
-          <View style={style.imageContainer}>
-            <Image
-              height={50}
-              width={50}
-              source={{uri: announcement.summaryImage}}
-            />
+    <Section style={style} key={announcement.id} testID="announcement">
+      <GenericSectionItem>
+        <View style={styles.container}>
+          <View
+            style={styles.content}
+            accessible={true}
+            accessibilityRole={announcement.actionButton ? 'button' : undefined}
+            accessibilityHint={
+              announcement.actionButton !== undefined
+                ? t(DashboardTexts.announcemens.button.accessibility)
+                : undefined
+            }
+          >
+            {announcement.summaryImage && (
+              <View style={styles.imageContainer}>
```

Comment:
I noticed that in the `imageContainer` style, there is a negative padding value set. I don't think that is valid and will have no effect, so it should be removed I think (see line 151)